### ISSUE TITLE
pathauto_update_8001() kills some entity.definitions.installed

### ISSUE THREAD

**Issue by Leksat** [md-systems/pathauto#126]
I guess this is something different from #117.

That what I faced after the pathauto_update_8001() update which was introduced in 69add3d:
<img width="877" alt="screen_shot_2016-01-09_at_05_51_45" src="https://cloud.githubusercontent.com/assets/989015/12214023/7ef44310-b696-11e5-9010-e4782db1045e.png">

This was not noticed right after the update because error reporting was off. For instance, here are some exceptions:

```
array_intersect_key(): Argument #2 is not an array content_translation.module:188
array_diff_key(): Argument #2 is not an array EntityDefinitionUpdateManager.php:289
Invalid argument supplied for foreach() EntityDefinitionUpdateManager.php:289
array_diff_key(): Argument #1 is not an array EntityDefinitionUpdateManager.php:294
Invalid argument supplied for foreach() EntityDefinitionUpdateManager.php:294
array_intersect_key(): Argument #2 is not an array EntityDefinitionUpdateManager.php:299
Invalid argument supplied for foreach() EntityDefinitionUpdateManager.php:299
```

Status report could not compare definitions and reported `Entity/field definitions: Up to date`.

I currently try to resolve this.

**Comment by Leksat** [comment]
Oh damn... This was fixed in 9d71dc7. Lucky me...

**Comment by Leksat** [comment]
Just in case if someone meet the issue, which is unlikely, here is how I fixed it:

``` php
$affected_entity_type_ids = ['node', 'taxonomy_term', 'user'];
/** @var \Drupal\Core\Entity\EntityTypeManager $etm */
$etm = \Drupal::service('entity_type.manager');
/** @var \Drupal\Core\Entity\EntityLastInstalledSchemaRepository $elisr */
$elisr = \Drupal::service('entity.last_installed_schema.repository');
/** @var \Drupal\Core\Entity\EntityFieldManager $efm */
$efm = \Drupal::service('entity_field.manager');
foreach ($affected_entity_type_ids as $entity_type_id) {
  $entity_type = $etm->getDefinition($entity_type_id);
  if ($entity_type->isSubclassOf(\Drupal\Core\Entity\FieldableEntityInterface::class)) {
    $elisr->setLastInstalledFieldStorageDefinitions($entity_type_id, $efm->getFieldStorageDefinitions($entity_type_id));
  }
}
\Drupal::entityDefinitionUpdateManager()->applyUpdates();
```
